homeassistant/components/lutron_caseta/manifest.json

```diff
@@ -9,7 +9,7 @@
   },
   "iot_class": "local_push",
   "loggers": ["pylutron_caseta"],
-  "requirements": ["pylutron-caseta==0.23.0"],
+  "requirements": ["pylutron-caseta==0.24.0"],
```

Comment:
Please bump the library in a separate PR

We should only have one box the `Type of change` boxes checked per PR.

Thanks!